src/gocept/selenium/tests/test_wd_selenese.py

```diff
@@ -205,9 +206,7 @@ def test_csscount_raises_nice_exception_on_mismatch(self):
 
     def test_configured_timeout_is_applied_for_open(self):
         self.selenium.setTimeout(1)
-        with self.assertRaisesRegexp(
-                # Chromium and Firefox use different messages.
-                Exception, 'Message: [T|t]imeout'):
```

Comment:
Did Chrome change the Exception also?

Reply:
No, it still uses this exception text but I thought it would be more sane to test for the actual exception instead of its text.